.github/workflows/merge-dependencies.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Merge Dependencies'
+
+# https://github.com/ahmadnassri/action-dependabot-auto-merge/issues/60#issuecomment-806027389
+on: [pull_request_target]
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    # Guarantee that commit comes from Dependabot (don't blindly trust external GitHub Actions)
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - name: 'Automerge dependency updates from Dependabot'
+        uses: ahmadnassri/action-dependabot-auto-merge@v2.4.0
+        with:
+          github-token: ${{ secrets.GH_AUTOMERGE_TOKEN }}
+          node-version: 12.x
```

Comment:
I don't think that works here, at least I don't see `node-version` as a property within [ahmadnassri/action-dependabot-auto-merge](https://github.com/ahmadnassri/action-dependabot-auto-merge)